src/config.rs

```diff
@@ -227,6 +257,10 @@ impl Config {
             })
         }
     }
+
+    pub fn index_settings(&self) -> Bytes {
+        bincode::serialize(&(self.txstore_enabled, self.blocktxs_enabled, self.blockmeta_enabled)).unwrap()
```

Comment:
Do we also want to have `prevout_enabled` and `network_type` as part of the settings?

Reply:
`prevout_enabled` does not effect the database structure, just the way data is queried. Users can freely change this without a reindex.

It could be useful to add `network_type`, to detect cases where the user changes a network but uses the same `db` dir. This is also relevant for upstream electrs.

Reply:
Added `network_type` to `index_settings()` in e8fd843.

Reply:
Thanks!